app/src/common/common.actions.js

```diff
@@ -28,11 +28,78 @@ export const getGraphQL = (userToken, query, variables, onSuccess, onError) => {
             return { errors: [ err ] }
         })
         .then(response => {
+            if (response.errors) {
+                onError(response.errors.map(err => err.message || err).join('. '))
```

Comment:
when calling a callback, you should prefer to `return cb();`. It ensures the code paths.